ci/do_ci.sh

```diff
@@ -1046,7 +1045,9 @@ case $CI_TARGET in
             "${CURRENT_SCRIPT_DIR}/../tools/proto_format/proto_format.sh" fix
         fi
 
-        "${CURRENT_SCRIPT_DIR}/../tools/gen_compilation_database.py" "${CI_PARAMS[@]}"
+        read -ra ENVOY_GEN_COMPDB_OPTIONS <<< "${ENVOY_GEN_COMPDB_OPTIONS:-}"
```

Comment:
you might want to add this env var to `ci/run_envoy_docker.sh`

Reply:
Good point. Done. Thanks.